guardian/interactive-atom-thrasher-template
# Replace The Fiver by Football Daily in set-7

## What does this change?

Update the multi thrasher in set-7, replacing The Fiver by Football Daily

## How to test

Run `gulp` from your ID
Select `set-7` from the list of Atoms then select one of the Thrasher containers or Snaps

To test the success/error message has inverted colours for the Football Daily thrasher, see the testing instructions in PR #39

## Have we considered potential risks?

This PR cannot be completed with urls to the new sign up pages until they are launched on 16 Nov 2022

## Images

Before
![Screenshot 2022-11-08 at 12 38 21](https://user-images.githubusercontent.com/74301289/200565652-b0229de4-ea4a-497d-b936-12142213263a.png)

After
![Screenshot 2022-11-08 at 12 36 30](https://user-images.githubusercontent.com/74301289/200565320-f98897b8-117f-48ca-8ed9-f68e86217acd.png)
